ert_shared/models/base_run_model.py

```diff
@@ -186,11 +197,44 @@ def get_runtime(self) -> Union[int, float]:
         else:
             return self.stop_time() - self.start_time()
 
+    def getQueueSize(self) -> int:
+        if self._job_queue is None:
+            return 1
+
+        queue_size = len(self._job_queue)
+
+        if queue_size == 0:
+            queue_size = 1
+
+        return queue_size
```

Comment:
Think something might have gone wrong in the rebase, as the functions are now reintroduced :thinking:

Reply:
This PR does not make that much sense anyway after job_queue was removed.